openSUSE-Leap-42.1/config.kiwi

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<image schemaversion="6.1" name="openSUSE-42.1-docker-guest">
+<image schemaversion="6.5" name="openSUSE-Leap-42.1-docker-guest">
   <description type="system">
     <author>Flavio Castelli</author>
     <contact>fcastelli@suse.com</contact>
```

Comment:
Given we now have a team, I propose to switch my personal details with the one of the team. Use the same data we use for the SLE images